tests/console-backend-service/internal/domain/eventing/trigger_test.go

```diff
@@ -208,7 +211,7 @@ func triggerDetailsFields() string {
 		name
     	namespace
 		broker
-    	filterAttributes
+		filterAttributes
```

Comment:
indentations in this whole query look weird

Reply:
fixed